axelor-human-resource/src/main/resources/views/Employee.xml

```diff
@@ -266,7 +265,11 @@
     <action-method name="action-employee-method-normalize-emergencyNumber">
         <call class="com.axelor.apps.base.web.PartnerController" method="normalizePhoneNumber"/>
     </action-method>
-
+    
+	<action-validate name="action-employee-validate-dates">
```

Comment:
Use action-condition instead of action-validate , check existing action-condition for reference.